src/networkpolicy/helperFunctions.go

```diff
@@ -796,8 +810,9 @@ func GetNetPolicy(cluster, namespace, policyType string) *wpb.WorkerResponse {
 
 			response.Ciliumpolicy = append(response.Ciliumpolicy, &ciliumpolicy)
 		}
-		response.K8SNetworkpolicy = nil
-	} else if strings.Contains(policyType, "NetworkPolicy") {
+
+	}
+	if strings.Contains(pType, "generic") {
```

Comment:
else if strings.Contains(pType, "generic") {

Reply:
@Ankurk99 We are planning to get both at same point if option permits. That is why we are using a **if** case instead of **else if**

Reply:
@Ankurk99 We are planning to get both at same point if option permits. That is why we are using a **if** case instead of **else if**

Reply:
No @Ankurk99 We support to display both at same time